epictrack-web/src/utils/index.ts

```diff
@@ -6,20 +6,16 @@ const sort = (collection: any[], sortField: string) => {
     ignorePunctuation: true,
     sensitivity: "base",
   });
-
   const keys = sortField.split(".");
-
-  return [...collection].sort(function (a, b) {
-    let aValue = a;
-    let bValue = b;
-
-    keys.forEach((key) => {
-      aValue = aValue[key];
-      bValue = bValue[key];
+  if (keys && keys.length === 0) {
+    return collection.sort(function (a, b) {
+      return collator.compare(a[sortField], b[sortField]);
     });
-
-    return collator.compare(aValue, bValue);
-  });
+  } else {
+    return collection.sort(function (a, b) {
+      return collator.compare(a[keys[0]][keys[1]], b[keys[0]][keys[1]]);

```

Comment:
nice, this works for our current needs. But I feel something like "key1.key2.key3" might break this, so maybe add a safety if statement if key.length > 2 for now and return something default

Reply:
this would be a technical debt to be solved later maybe, by recursion probably

Reply:
Lets assume we would go only two nested levels in the json for now